packages/style-engine/src/styles/elements/index.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+// @ts-ignore (Reason: Could not find a declaration file for module '@wordpress/blocks)
+import { __EXPERIMENTAL_ELEMENTS as ELEMENTS } from '@wordpress/blocks';
```

Comment:
What do you think about bringing the list of elements into the style engine, so that we don't have to create this additional dependency? We're already using the style engine as its own source of truth for the shape of all the other properties, so I was wondering if it would make sense for it to have its own constant for the list of elements it deals with?

Reply:
That's a good idea!

There's a pretty massive TODO on this PR. To look at `packages/edit-site/src/components/global-styles/use-global-styles-output.js`.

That's the other place where `ELEMENTS` are imported. 

So once I deal with that, I think we should be golden.

Reply:
Ah, gotcha, that's a good point. Thanks for following up!